src/k6/aws-cli-objects.js

```diff
@@ -24,12 +25,23 @@ export function setup() {
 }
 
 export function teardown({bucketName}) {
+  // list objects
+  let stdOut = s3api("list-objects", bucketName, [])
+  let contents = parse(stdOut).Contents
+  let objects = contents.map(o => {
+    return {"Key": o.Key}
+  })
+  // delete objects
+  stdOut = s3api("delete-objects", bucketName, [
+    "--delete", JSON.stringify({Objects: objects})
+  ])
```

Comment:
daria para usar o aws rb --force tambem `aws s3 rb s3://bucket-name --force` se quiser simplificar

Reply:
Boa!

Reply:
`rb` é operador do comando `s3`. Até agora só foi usado comando `s3api`. Tudo bem misturar os dois comandos nos testes?
Ref: https://aws.amazon.com/pt/blogs/developer/leveraging-the-s3-and-s3api-commands/

Reply:
acho que tudo bem, é aws-cli e este nao é um teste do rb, é só uma limpeza do teardown